src/main/msp/msp.c

```diff
@@ -2492,6 +2492,33 @@ static mspResult_e mspFcProcessOutCommandWithArg(mspDescriptor_t srcDesc, int16_
         }
 
         break;
+
+    case MSP2_GET_TEXT:
+        {
+            // type byte, then length byte followed by the actual characters
+            const uint8_t textType = sbufBytesRemaining(src) ? sbufReadU8(src) : 0;
+
+            char* textVar;
+            uint8_t textLength;
+            switch (textType) {
+                case MSP2TEXT_PILOT_NAME:
+                    textVar = pilotConfigMutable()->pilotName;
+                    textLength = strlen(textVar);
+                    break;
+
+                default:
+                    return MSP_RESULT_ERROR;
+            }
+
+            //  type byte, then length byte followed by the actual characters
+            sbufWriteU8(dst, textType);
+            sbufWriteU8(dst, textLength);
+            for (int i = 0; i < textLength; i++) {
```

Comment:
unsigned int?

Reply:
Thanks for spotting this! I also pushed some minor syntactical improvements across both methods.